addons/centered/src/index.js

```diff
@@ -9,8 +9,13 @@ const style = {
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
+  overflow: 'auto',
 };
 
+const innerStyle = {
+  margin: 'auto',
+}
+
 export default function(storyFn) {
-  return <div style={style}>{storyFn()}</div>;
+  return <div style={style}><div style={innerStyle}>{storyFn()}</div></div>;
```

Comment:
I think adding another container defeats the purpose of the centered addon, I'll have to test.